.azure-pipelines/beta-build-production.yml

```diff
@@ -0,0 +1,30 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  - main
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+
+- script: git checkout main
+- task: CopyFiles@2
+  displayName: 'Copy Files to staging directory'
+  inputs:
+    SourceFolder: '$(System.DefaultWorkingDirectory)'
+    Contents: |
+     **/*
+     !spec/**
+     !.azure-pipelines/**
+     !scripts/**
+     !.github/**
+     !.git/**
+     !.vscode/**
```

Comment:
Do you want to keep typings-demo.gif in the copy?

Reply:
Added this in the contents to be skipped.